caikit/runtime/model_management/model_manager.py

```diff
@@ -197,9 +197,16 @@ def load_model(
                         raise ex
 
                     # Estimate the model's size and update the LoadedModel
-                    model_size = self.model_sizer.get_model_size(
-                        model_id, local_model_path, model_type
-                    )
+                    try:
+                        model_size = self.model_sizer.get_model_size(
+                            model_id, local_model_path, model_type
+                        )
+                    except CaikitRuntimeException:
+                        log.debug(
```

Comment:
hmm.. is this the only case when `get_model_size` would fail with runtime exception?

Reply:
Yeah, it wraps all of its error handling with `CaikitRuntimeException`. If there are other unhandled error cases, they're bugs, so we would want them to surface anyway.